PracticeGit/intlist.c

```diff
@@ -0,0 +1,152 @@
+//----------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+//------------------------------------------------------------------------------------------------

```

Comment:
Please format your code. Your tab is 4 spaces long, it's supposed to be 3 spaces

Reply:
it has 3 spaces only